jumanji/types.py

```diff
@@ -95,6 +95,7 @@ def restart(
     observation: Observation,
     extras: Optional[Dict] = None,
     shape: Union[int, Sequence[int]] = (),
+    dtype: Union[jnp.dtype, type] = float,
```

Comment:
Can you please add this to the docs for all these methods :pray:

Reply:
Oh mb, I forgot about it, it's done, tell me if it is okay :)